support/mock/builder.go

```diff
@@ -80,3 +81,13 @@ func (b *RuntimeBuilder) WithBalance(balance, received abi.TokenAmount) *Runtime
 	b.rt.valueReceived = received
 	return b
 }
+
+func (b *RuntimeBuilder) WithReceiverType(code cid.Cid) *RuntimeBuilder {
+	b.rt.actorCodeCIDs[b.rt.receiver] = code
+	return b
+}
+
+func (b *RuntimeBuilder) WithSysCalls(calls runtime.Syscalls) *RuntimeBuilder {
+	b.rt.syscalls = calls
+	return b
```

Comment:
Rather than this, I'd rather add `WithVerifiesSig` here, or `SetVerifiesSig` on the mock that update individual syscall stubs. The mock syscalls object is great, but hide it from the user.

Reply:
@anorth  The thing I ended up doing (the simpler thing) was to make a syscaller struct but it's not terribly elegant and there is duplication.  I could do some reflection magic and reduce some duplication.  Let me know what your preference is, or if you have another suggestion.